falcon/media/handlers.py

```diff
@@ -41,7 +47,7 @@ class Handlers(UserDict):
     def __init__(self, initial=None):
         self._resolve = self._create_resolver()
 
-        handlers = initial or {
+        handlers: Mapping[str, Serializer] = initial or {
```

Comment:
Why do we need to define `Serializer` (probably not the best name since it also does deserialization, not only serialization), cannot we just type it what it is, `base.Handler`?